test/config/ca.json

```diff
@@ -133,7 +133,11 @@
     "challenges": {
       "http-01": true,
       "tls-sni-01": true,
-      "dns-01": true
+      "dns-01": true,
+      "tls-sni-02": true
+    },
+    "features": {
+      "AllowTLS02Challenges": true
```

Comment:
This feature flag should only be enabled in the `config-next/` json files, not `config/` so the default flag state is tested except when BOULDER_CONFIG_DIR is set to config-next.